test/jdk/java/awt/Dialog/FileDialogIconTest/FileDialogIconTest.java

```diff
@@ -55,15 +56,24 @@ public class FileDialogIconTest {
 
     public static void main(String[] args) throws Exception {
         String INSTRUCTIONS = """
-                1. Select the Image for a Dialog and Frame using either
-                   Load/Save/Just Dialog.
-                2. Set the Icon Image/s to Frame and Dialog. Verify that the
-                   Icon is set for the respective Frame and Dialog.
-                   If selected Icon is set to Frame and Dialog press PASS
-                   else FAIL.
-                                """;
+                The 1st row of buttons in testUI are to open Load,
```

Comment:
It may not be apparent what `testUI` is… on the other hand, it should still be obvious that you referring to a window on the right.

It could better to refer to it as a ‘window’… or a least ‘test UI’ (with a space).